scripts/ci/ci_run_airflow_testing.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+#
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+set -xeuo pipefail
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+export PYTHON_VERSION=${PYTHON_VERSION:=$(python -c 'import sys; print("%s.%s" % (sys.version_info.major, sys.version_info.minor))')}
+export AIRFLOW_VERSION=$(cat airflow/version.py | grep version | awk {'print $3'} | tr -d "'+")
+export DOCKERHUB_USER=${DOCKERHUB_USER:="potiuk"} # TODO: change back to "airflow"
```

Comment:
I noticed this as well:
```
MacBook-Pro-van-Fokko:incubator-airflow fokkodriesprong$ docker ps
CONTAINER ID        IMAGE                                      COMMAND                  CREATED             STATUS              PORTS                                         NAMES
c556cab3143e        potiuk/airflow:latest-3.6-ci-v2.0.0.dev0   "/bin/bash -c '/opt/…"   6 minutes ago       Up 6 minutes        0.0.0.0:28080->8080/tcp                       ci_airflow-testing_run_dfff807d8d98
563729cd183f        rabbitmq:3.7                               "docker-entrypoint.s…"   6 minutes ago       Up 6 minutes        4369/tcp, 5671-5672/tcp, 25672/tcp            ci_rabbitmq_1
e04e043b8cf7        cassandra:3.0                              "docker-entrypoint.s…"   6 minutes ago       Up 6 minutes        7000-7001/tcp, 7199/tcp, 9042/tcp, 9160/tcp   ci_cassandra_1
2fd6654702cd        postgres:9.6                               "docker-entrypoint.s…"   6 minutes ago       Up 6 minutes        0.0.0.0:25433->5432/tcp                       ci_postgres_1
f41320c92c7b        godatadriven/krb5-kdc-server               "/bin/bash /app/star…"   6 minutes ago       Up 6 minutes                                                      ci_krb5-kdc-server_1
0dda7f9a6e3e        mysql:5.6                                  "docker-entrypoint.s…"   6 minutes ago       Up 6 minutes        0.0.0.0:23306->3306/tcp                       ci_mysql_1
7c1dd4bb3448        osixia/openldap:1.2.0                      "/container/tool/run…"   6 minutes ago       Up 6 minutes        389/tcp, 636/tcp                              ci_openldap_1
5a7b4a387b0b        redis:5.0.1                                "docker-entrypoint.s…"   6 minutes ago       Up 6 minutes        6379/tcp                                      ci_redis_1
76f435ff6386        mongo:3                                    "docker-entrypoint.s…"   6 minutes ago       Up 6 minutes        27017/tcp                                     ci_mongo_1
```

Reply:
Yeah. For now I keep "potiuk" in there because I can run testing with my own DockerHub/Travis (but when we get to merging/splitting to separate commits I will bring it back to "airflow" (there is a TODO in configuration). Actually this is also a feature requested in "Simplified Development Workflow" - you will be able to use your own DockerHub account/DockerHub repo to build and test (and use) images on your own instance before it ever hits DockerHub or TravisCI of airflow. Kinda nice for testing.